marklogic-data-hub-central/ui/src/components/modeling/property-modal/property-modal.tsx

```diff
@@ -63,7 +63,7 @@ const ALL_RADIO_DISPLAY_VALUES = [
 const ALL_CHECKBOX_DISPLAY_VALUES = [
```

Comment:
There are no unit test updates to the property-modal.

Reply:
Added and updated unit tests in property-modal.test.tsx